src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs

```diff
@@ -38,7 +38,7 @@ public class LanguageServer
         /// <summary>
         /// This const represents the dummy formula that is used to create an infrastructure needed to get the symbols for Nl2Fx operation.
         /// </summary>
-        internal static readonly string Nl2FxDummyFormula = "\"f7979178-07f0-424d-8f8b-00fee6fd19b8\"";
+        public static readonly string Nl2FxDummyFormula = "\"f7979178-07f0-424d-8f8b-00fee6fd19b8\"";
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

why does this need to be public?

Doc server has internals access.

Reply:
I do not seem to be able to access this in doc server. It doesn't even show up in intellisense. Let me look more into it tho.

Reply:
LSP assembly didn't made its internals visible to DocumentServer.Core. I added that change and changed this back to internal